components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -152,6 +152,20 @@ sealed class TabListAction : BrowserAction() {
      */
     data class AddMultipleTabsAction(val tabs: List<TabSessionState>) : TabListAction()
 
+    /**
+     * Moves a set of tabIDs into a new position (maintaining internal ordering)
+     *
+     * @property tabIds The IDs of the tabs to move.
+     * @property targetTabId A tab that the moved tabs will be moved next to
+     * @property placeAfter True if the moved tabs should be placed after the target,
+     * False for placing before the target. Irrelevant if the target is one of the tabs being moved.
```

Comment:
Puts the list of tabs next to where the target tab was. In this scenario placeAfter doesn't matter, since before and after the target are the same place. I will add this information to the documentation.